app/backend/src/clj/teet/cooperation/cooperation_db.clj

```diff
@@ -179,16 +179,20 @@
   "Given project-id and an application with a date, return an activities id that is on going during the dates"
   [db project-id application]
   (let [application-date (:cooperation.application/date application)]
-    (->> (:thk.project/lifecycles
-           (d/pull db '[{:thk.project/lifecycles
-                         [{:thk.lifecycle/activities
-                           [:db/id
-                            :activity/name
-                            :activity/estimated-start-date
-                            :activity/estimated-end-date]}]}]
-                   [:thk.project/id project-id]))
-         (mapcat
-           :thk.lifecycle/activities)
+    (->> (mapv
+           first
+           (d/q '[:find (pull ?ac [:db/id
+                                   :meta/deleted?
```

Comment:
why pull `:meta/deleted?` if you are checking that it is missing below... it can't have value.

Reply:
True it was left over from debugging.